src/commands/optimize.ts

```diff
@@ -178,6 +190,11 @@ export default class Optimize extends Command {
       this.showOptimizations(report.reuseComponents);
       choices.push({name: 'reuse components', value: Optimizations.REUSE_COMPONENTS});
     }
+    if (canIgnoreSchema) {
+      this.log("Do not ignore schema for the components");
+      choices.push({name: 'Do not ignore schema', value: DisableOptimizations.SCHEMA});
```

Comment:
I suggest we use the same option as flag. I mean the name should be `Ignore schemas`. Also it should be checked by default if `--ignore=schema` is passed.